spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do
+  describe '#FacilityFactory' do
+    it 'can initialize and object' do
+      factory = FacilityFactory.new
+      expect(factory).to be_an_instance_of(FacilityFactory)
+    end
+  end
+
+  describe '#create facilities' do
+    it 'can create a facility from provided data' do
+      factory = FacilityFactory.new
+      co_dmv_office_locations = DmvDataService.new.co_dmv_office_locations
+      result = factory.create_facilities(co_dmv_office_locations)
+      expect(result.class).to eq(Array)
+      expect(result.first.class).to eq(Facility)
```

Comment:
I think it would be valuable to also `expect(result.last.class).to eq(Facility)`. Alternatively:
```
result.each do |facility|
  expect(facility).to be_instance_of(Facility)
  # could test the data types of each Facility instance's attributes here, too
end
```